change/@ni-nimble-components-27f5d5a2-511b-45fd-991a-bee83da87ed5.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think continued review of the API in this spec contingent based on discussions in: https://dev.azure.com/ni/DevCentral/_workitems/edit/2383256

So I'm moving the PR to draft per our current contribution guidelines until that discussion is resolved. Let me know if there are any concerns

Reply:
@vikisekarNI I see you moved this PR out of draft and back to "ready for review" but I still see several open comment threads. Could you please close those discussions with a short summary of what was decided or what input you still need from reviewers?

Reply:
Thanks for the heads up @jattasNI. I will respond with decisions we made in our WCAG call for link support on open comments. I will also re-enable draft mode, since we planned to do prototyping for below comment from Milan. We'll mark it ready for review, once we have the feasibility/outcome of prototype.

![image](https://github.com/ni/nimble/assets/94439533/ec3e9b29-84b1-4bb6-aa97-c897e99d106e)